events/bounty-hunters.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bounty Hunters
+---
+
+Bounty Hunters este un nou eveniment permanent activ pe server, vizibil în `/event` similar cu World Exploration. In fiecare zi, serverul va spawna pentru fiecare jucator la un interval de 8 ore (00:00, 08:00, 16:00) un numar de NPC-uri ce trebuiesc ucise.
+In functie de tipul contului detinut se vor spawna 8 NPC-uri (cont standard & Gold) sau 10 NPC-uri (cont Platinum).
+
+Exista 3 tip-uri de NPC-uri:
+"Peaceful" - nu risposteaza cand este atacat
+"Aggressive" - riposteaza cand este atacat
+"Aggressive with suitcase" - riposteaza cand e atacat si dropeaza o servieta la deces, servieta ce trebuie livrata la Alfred intr-un interval de 10 minute
+
+Evenimentul dispune de un sistem de avansare in nivel, avansarea in nivel oferind jucatorilor premimii mai atractive.
+	
+In momentul in care jucatorii omoara un NPC sau livrezi servieta vor primi urmatoarele premii:
+- intre 1 si 3 Streep Rep
+- 5 Heat Level
+- 50 Experienta (+ bonus in functie de nivelul Bounty Hunters) (+20% daca jucatorii beneficiaza de cont Platinum)
+- $800-$1200 bani marcati
+- o sansa de 25-45% (in functie de nivelul de Bounty Hunters) de a dropa **unul** dintre urmatoarele iteme:
+    - Reel Rod
+    - Rod Grip
+    - Rod Guide
+	- Plastic
+	- Metal
+	- Gunpowder
+	- Glass
+	- Wires
+	- Electronic Keyboard
+	- Explosive
+	- Drill
+	- Pliers
+	
```

Comment:
**Ce primesc in cadrul evenimentului "Bounty Hunters"**

In momentul in care omori un NPC sau livrezi servieta vei primi urmatoarele premii: intre 1 si 3 Street Rep, 5 Heat Level, 50 Experienta(+ un bonus in functie de nivelul de Bounty Hunters); (+20% daca detii o subscriptie Platinum asociata contului tau), 800$ - 1.200$ bani marcati si o sansa de 25% - 45%(in functie de nivelul de Bounty Hunters) de a dropa unul dintre urmatoarele iteme: Rod Guide, Rod Grip, Reel Rod, Metalbar, Gunpowder, Shattered Glass, Wires, Electronic Keyboard, Explosive, Drill & Pliers.